sigmf/schema-meta.json

```diff
@@ -71,6 +71,21 @@
                 "required": false,
                 "help": "Information about the hardware used (measurement setup, antennas, etc.)"
             },
+            "core:dataset": {
+                "type": "string",
+                "required": false,
+                "help": "The full filename of the dataset file this Metadata file describes."
+            },
+            "core:trailing_bytes": {
+                "type": "uint",
+                "required": false,
+                "help": "The number of bytes to ignore at the end of a Non-Conforming Dataset file."
+            },
+            "core:virtual_dataset": {
+                "type": "bool",
+                "required": false,
+                "help": "Always `true`, indicates that the dataset is virtual (SigMF Recording is metadata only)."
```

Comment:
The `Always true` part of this is confusing me, I think?

Reply:
I think i am going to change this. basically the idea was "only use this key if this is a virtual dataset, therefore if its defined it should be true". this is unnecessarily restrictive though and when using metadata only sigmf as a generator, it may be desirable to simply redefine this field to `false` after generating the dataset.